UI/src/view/changeInitialViewPopup.js

```diff
@@ -0,0 +1,215 @@
+window.ChangeInitialViewPopup = function(backend, location, userRole, places, assetTypeConfig, defaultAsset, municipality) {
+
+  var options = {
+    confirmButton: 'Tallenna',
+    cancelButton: 'Peruuta',
+    successCallback: function () {
+      var selectedLocation = $('#location').val();
+      var selectedAssetType = parseInt($('#assetType').val(), 10);
```

Comment:
please, create a constant and replace all place where you are expecting  massTransitStopId